nautobot_device_lifecycle_mgmt/models.py

```diff
@@ -215,13 +230,96 @@ def to_csv(self):
             self.release_date,
             self.end_of_support,
             self.documentation_url,
-            self.download_url,
-            self.image_file_name,
-            self.image_file_checksum,
             self.long_term_support,
             self.pre_release,
         )
 
+    objects = SoftwareLCMQuerySet.as_manager()
```

Comment:
is this necessary? (I though it was inherited)

Reply:
This is a custom QuerySet that adds `get_for_object` method. Default QuerySet/Manager is `RestrictedQuerySet` which comes from `BaseModel`  class:

https://github.com/nautobot/nautobot/blob/develop/nautobot/core/models/__init__.py#L28

```
objects = RestrictedQuerySet.as_manager()
```